docs/src/main/sphinx/release/release-398.md

```diff
@@ -0,0 +1,61 @@
+# Release 398 (28 Sep 2022)
+
+## General
+
+* Add Hudi connector. ({issue}`10228`)
+* Add metrics for the execution time of filters and projections to `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`14135`)
+* Show local cost estimates when using `EXPLAIN`. ({issue}`14268`)
+* Fix timeouts happening too early because of improper handling of the
+  `node-scheduler.allowed-no-matching-node-period` configuration property. ({issue}`14256`)
+* Fix query failure for `MERGE` queries when `task_writer_count` is greater
+  than one. ({issue}`14306`)
+
+## Accumulo connector
+
+* Add support for column comments when creating a new table. ({issue}`14114`)
+* Add column mapping and index information in an extra field instead of a
```

Comment:
What's an "extra field"?

Reply:
`Extra` column of `DESCRIBE` statement result to be exact.